TimeCircuitViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  TimeCircuitViewController.swift
+//  outtatime
+//
+//  Created by Lambda_School_Loaner_268 on 1/29/20.
+//  Copyright © 2020 Jeremiah. All rights reserved.
+//
+
+
+
+import UIKit
+
+class TimeCircuitViewController: UIViewController, DatePickerDelegate {
+    
+    var timer: Timer = Timer()
+    
+    var destinationDate: Date?
+    func destinationDateWasChosen(date: Date) {
+        destinationDate = date
+        destinationTime.text = dateFormatter.string(from: date)
+        
+    }
+    
+    func startTimer() {
+        timer = Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true, block: updateSpeed(timer:)
+)
+        
+    }
+    func resetTimer() {
+        timer.invalidate()
+    }
+    func updateSpeed(timer: Timer) {
+       while speed < 88 {
+           speed += 1
+           speedLabel.text = "\(speed) MPH"
+       }
+     
+        // stopTimer()
+        resetTimer()
+           lastTimeDeparted.text = presentTime.text
+           presentTime.text = destinationTime.text
+           speed = 0
+           speedLabel.text = "0 MPH"
+           showAlert()
+       
+   }
+
+    
+    
+    
+    func showAlert() {
+        let alert = UIAlertController(title: "Time Travel Successful", message: "Your new date is \(presentTime.text!).", preferredStyle:  .alert)
+        let okAction = UIAlertAction(title: "Time Travel Successful", style: .default, handler: nil)
```

Comment:
Instead of creating a separate alert action here add it to the alert you already created in line 52.
EX: `alert.addAction(title: "Time Travel Successful", style: .default, handler: nil)`